### PR TITLE
Feature/add new smoke doc tests

### PR DESCRIPTION
**Pull request by flaminestone**

### CODE REVIEW

File: spec/docx/smoke/api_table_row_properties_spec.rb [hunk 1]
```diff
@@ -2,15 +2,18 @@
 describe 'ApiTableRowPr section tests' do
   it 'ApiTableRowPr | GetClassType method' do
     docx = DocBuilderWrapper.new.build_doc_and_parse('asserts/js/docx/smoke/ApiTableRowPr/getclasstype.js')
-    expect(docx.nil?).to eq(false)
+    expect(docx.elements.first.character_style_array[1].text).to eq('Class Type = tableRowPr')
   end
 
   it 'ApiTableRowPr | SetHeight method' do
+    skip 'Parser error https://github.com/ONLYOFFICE/ooxml_parser/issues/263'
```
Comment (ShockwaveNN) [inline spec/docx/smoke/api_table_row_properties_spec.rb]: https://github.com/ONLYOFFICE/ooxml_parser/issues/263 is fixed

File: spec/docx/smoke/api_table_row_properties_spec.rb [hunk 2]
```diff
@@ -2,15 +2,18 @@
...
     docx = DocBuilderWrapper.new.build_doc_and_parse('asserts/js/docx/smoke/ApiTableRowPr/setheight.js')
-    expect(docx.nil?).to eq(false)
+    expect(docx.elements[1].properties.table_style.table_row_properties.height.value).to eq(OoxmlParser::OoxmlSize.new(720, twip))
+    expect(docx.elements[1].properties.table_style.table_row_properties.height.type).to eq('atLeast')
   end
 
   it 'ApiTableRowPr | SetTableHeader method' do
+    skip 'Parser error https://github.com/ONLYOFFICE/ooxml_parser/issues/264'
```
Comment (ShockwaveNN) [inline spec/docx/smoke/api_table_row_properties_spec.rb]: https://github.com/ONLYOFFICE/ooxml_parser/issues/264 is fixed

File: spec/docx/smoke/api_table_row_spec.rb [hunk 3]
```diff
@@ -2,25 +2,27 @@
...
   end
 
   it 'ApiTableRow | GetClassType method' do
     docx = DocBuilderWrapper.new.build_doc_and_parse('asserts/js/docx/smoke/ApiTableRow/getclasstype.js')
-    expect(docx.nil?).to eq(false)
+    expect(docx.elements.first.character_style_array[1].text).to eq('Class Type = tableRow')
   end
 
   it 'ApiTableRow | SetHeight method' do
+    skip 'Parser error https://github.com/ONLYOFFICE/ooxml_parser/issues/263'
```
Comment (ShockwaveNN) [inline spec/docx/smoke/api_table_row_spec.rb]: fixed

File: spec/docx/smoke/api_table_row_spec.rb [hunk 4]
```diff
@@ -2,25 +2,27 @@
...
     docx = DocBuilderWrapper.new.build_doc_and_parse('asserts/js/docx/smoke/ApiTableRow/setheight.js')
     expect(docx.nil?).to eq(false)
   end
 
   it 'ApiTableRow | SetTableHeader method' do
+    skip 'Parser error https://github.com/ONLYOFFICE/ooxml_parser/issues/264'
```
Comment (ShockwaveNN) [inline spec/docx/smoke/api_table_row_spec.rb]: fixed

File: spec/docx/smoke/api_table_row_properties_spec.rb (hunk 1 above)
Comment (flaminestone) [inline spec/docx/smoke/api_table_row_properties_spec.rb]: fixed

File: spec/docx/smoke/api_table_row_properties_spec.rb (hunk 2 above)
Comment (flaminestone) [inline spec/docx/smoke/api_table_row_properties_spec.rb]: fixed

File: spec/docx/smoke/api_table_row_spec.rb (hunk 3 above)
Comment (flaminestone) [inline spec/docx/smoke/api_table_row_spec.rb]: fixed

File: spec/docx/smoke/api_table_row_spec.rb (hunk 4 above)
Comment (flaminestone) [inline spec/docx/smoke/api_table_row_spec.rb]: fixed

**Pull request merged**